CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 Notable changes to QMCPACK will be documented in this file. 
 
+## [3.1.1] - 2017-08-01
+
+### Notes
+
+This is a bugfix release and recommended update.
+
+### QMCPACK updates
+
+* Added numerical tolerance to check of jastrow cutoff and Wiger Seitz
+  radius.
+* Cmake correctly configures when MPI is not present.
+* Improved support for test coverage measurements.
+* Added unit tests for some estimators.
+
+### NEXUS
+
+* ipython compatibile exit handling (from Duy Le)
```

Comment:
Spelling of compatible and capitilization of IPython.
"IPython-compatible exit handling (from Duy Le)"